test/app.integration.spec.js

```diff
@@ -154,4 +154,38 @@ describe('Test Activities:', () => {
         done();
       });
   });
-});
\ No newline at end of file
+});
+
+describe('Test Activities_Products', () => {
+  it('GET / All Activities_Products', (done) => {
+    request(app)
+      .get('/activitiesproducts')
+      .expect(200)
+      .expect('Content-Type', /json/)
+      .then(response => {
+        done();
+      });
+  });
+  it('GET / bad ID', (done) => {
+    request(app)
+      .get('/activitiesproducts/noId')
+      .expect(400)
```

Comment:
this should be a 404

Reply:
404 c'est quand l'id n'est pas trouvé, mais ici c'est une mauvaise requête donc 400 non ?